lib/service/film_manager.dart

```diff
@@ -0,0 +1,35 @@
+import 'dart:async';
+import 'package:my_movies/model/film.dart';
+import 'package:my_movies/widget/rating_button.dart';
+import 'package:rxdart/rxdart.dart';
+
+class FilmManager {
+  final _filmStreamController = StreamController<List<Film>>.broadcast();
+
+  List<Film> films = [];
+
+  Stream<List<Film>> get filmStream => _filmStreamController.stream
+      .startWith([]);
+
+  void addFilm(Film film) {
+    films.add(film);
+    _filmStreamController.sink.add(films);
+  }
+
+  void updateFilm(int index, Film updatedFilm) {
```

Comment:
Dával bych si pozor na updatování filmu podle indexu, protože musíš 100% zarušit, že tento index bude stejný jako index ve filmech. Vzniká ti zde pak chyba (ta pak dále v review)

Doporučil bych si spíše předávat jen Film a dohledat si jej podle ID nebo něčeho takového